typings/index.d.ts

```diff
@@ -1,12 +1,12 @@
 declare enum ChannelType {
-	TEXT = 'text',
-	DM = 'dm',
-	VOICE = 'voice',
-	GROUP = 'group',
-	CATEGORY = 'category',
-	NEWS = 'news',
-	STORE = 'store',
-	UNKNOWN = 'unknown'
+	text = 0,
+	dm = 1,
+	voice = 2,
+	group = 3,
+	category = 4,
+	news = 5,
+	store = 6,
+	unknown = 7
```

Comment:
You don't have to write the numbers here, the default behavior is to increment by one starting at 0.

Reply:
Just noticed too, on it